redisinsight/ui/src/components/global-url-handler/GlobalUrlHandler.tsx

```diff
@@ -67,13 +70,19 @@ const GlobalUrlHandler = () => {
     try {
       const params = new URLSearchParams(search)
       const from = params.get('from')
+      const returnUrl = params.get('returnUrl')
 
       if (from) {
         dispatch(setFromUrl(from))
         history.replace({
           search: ''
         })
       }
+      if (returnUrl) {
+        localStorageService.set(AppStorageItem.returnUrl, { value: returnUrl })
```

Comment:
`{value: ....}` is not needed, just 
`localStorageService.set(AppStorageItem.returnUrl, returnUrl)`